### DISCUSSION TITLE
Legacy skills [Remove]

### DISCUSSION THREAD

**Discussion by Dionis404** [sunflower-land/sunflower-land#3837] [Ideas]
I've noticed that new players, after playing for a couple of months and figuring out the game, start looking to buy farms with Legacy skills. 

Maybe we should remove Legacy skills from a farm after it is sold?

I think this would probably stabilize the situation with farm resale and re-verification problems. 
Yes, I realize that such actions are contrary to WEB3 principles.

**Comment by TheDcol** [comment]
It's an interesting approach, personally I wouldn't mind. . . 

What do you think instead, providing an option to buy an item that costs resources/sfl BUT doesnt stack with current legacy skills?

**Reply by Dionis404** [reply]
The idea is interesting, perhaps it will reduce the number of sales of old farms and players creating a farm will not think about what would buy legacy skills.

**Reply by TheDcol** [reply]
This way, OGs still have the bonus skill acquired at launch (and enjoyed it for 1, 2+ years) and newer players or those that minted a farm post beta can get that via a fee, without having to ditch their farm

**Reply by Dionis404** [reply]
Will players still have a choice between skills for crops and skills for animals? 
And what to do with those skills for resources? (I think they are extremely rare and they should not be added).

**Reply by brian-man** [reply]
@TheDcol Instead of introducing a new item, how about adding a new way to earn the existing legacy skill? Meaning convert it from "legacy" to "early access mints" and "current mints".

**Reply by TheDcol** [reply]
Earn it via achievement for example?
'harvest x many crops' (poor example but)

**Reply by brian-man** [reply]
Yes.  The original ones were earned.  Find an appropriate equivalent for the "modern" game and connect accordingly.
